crates/ark/src/lsp/symbols.rs

```diff
@@ -120,13 +120,56 @@ fn is_indexable(node: &Node) -> bool {
     true
 }
 
+// Function to parse a comment and return the section level and title
+fn parse_comment_as_section(comment: &str) -> Option<(usize, String)> {
+    // Match lines starting with one or more '#' followed by some non-empty content and must end with 4 or more '-' or '#'
+    // Ensure that there's actual content between the start and the trailing symbols.
+    let comment_re = regex::Regex::new(r"^(#+)\s*([^\s#-].+?)\s*(#{4,}|-{4,}|={4,})$").unwrap();
```

Comment:
```suggestion
    let comment_re = regex::Regex::new(r"^(#+)\s*([^\s#-=].+?)\s*(#{4,}|-{4,}|={4,})$").unwrap();
```

Right? Since 4+ `=` are also allowed as the section signal?